### ISSUE TITLE
After ComfyUI update StableSR always said Out of Memory

### ISSUE THREAD

**Issue by wardensc2** [gameltb/Comfyui-StableSR#7]
Hi gameltb

Can you check StableSR please, after ComfyUI update I can't upscale anything. Even upscale from 1024x1024 to 2048x2048, before update I can do with ease but now I can't do anything. My VGA is 3090 by the way

Thank you for supporting us

**Comment by light-and-ray** [comment]
I'm also getting OOM errors, but I remember using tiled diffusion in A1111. So maybe it will help, I'm trying

**Comment by light-and-ray** [comment]
Nope, it doesn't work. I remember StableSR extension in A1111 supported tiled diffusion, but the nodes here don't allow it
